scipy/special/tests/test_softmax.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import division
+
+import scipy.special as sc
+import numpy as np
+from numpy.testing import assert_almost_equal, assert_equal
+
+
+def test_softmax_fixtures():
+    assert_equal(sc.softmax([1000, 0, 0, 0]), np.array([1, 0, 0, 0]))
```

Comment:
Minor: Assert_equal in floating point?

Reply:
works, but agree should be `assert_almost_equal`